webapp/channels/src/components/channel_header_menu/menu_items/channel_bookmarks_submenu.tsx

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// Purpose of this file to exists is only required until channel header dropdown is migrated to new menus
+import React, {memo} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import {
+    ChevronRightIcon,
+    LinkVariantIcon,
+    PaperclipIcon,
+    BookmarkOutlineIcon,
+} from '@mattermost/compass-icons/components';
+import type {Channel} from '@mattermost/types/channels';
+
+import {useBookmarkAddActions} from 'components/channel_bookmarks/channel_bookmarks_menu';
+import * as Menu from 'components/menu';
+
+type Props = {
+    channel: Channel;
+};
+
+const ChannelBookmarksSubmenu = (props: Props) => {
+    const {formatMessage} = useIntl();
+
+    const {handleCreateLink, handleCreateFile} = useBookmarkAddActions(props.channel.id);
+    const canAdd = true; //useChannelBookmarkPermission(props.channel.id, 'add');
+    const canUploadFiles = true; //useCanUploadFiles();
+    const limitReached = false;
+
+    // useSelector((state: GlobalState) => {
+    //     const bookmarks = getChannelBookmarks(state, props.channel.id);
+    //     return bookmarks && Object.keys(bookmarks).length >= MAX_BOOKMARKS_PER_CHANNEL;
+    // });
```

Comment:
It looks like this got checked in by mistake

Reply:
Yes...thank you...updated to use selectors.